sdk/keyvault/azure-keyvault-secrets/tests/test_samples_secrets_async.py

```diff
@@ -3,14 +3,19 @@
 # Licensed under the MIT License.
 # -------------------------------------
 import asyncio
+import functools
```

Comment:
nit: unused import (I'm gonna stop repeating this comment, I'm sorry 😆)

Reply:
zapped em